docs-chef-io/content/inspec/resources/command.md

```diff
@@ -136,6 +136,28 @@ Wix includes several tools -- such as `candle` (preprocesses and compiles source
       end
     end
 
+### Timing Out Long-Running Commands
+
+On target platforms that support the feature, the command resource takes an option `timeout:` which specifies how long the command may run in seconds before erroring out and failing the control.
+
+    describe command("find / -owner badguy", timeout: 300) do
+      its("stdout") { should be_empty }
+    end
```

Comment:
````suggestion
```ruby
describe command("find / -owner badguy", timeout: 300) do
  its("stdout") { should be_empty }
end
```
````